newmiddleware/auth.js

```diff
@@ -31,320 +26,7 @@ class AuthMiddleware {
     })
     req.accessToken = accessToken
     if (userId) req.userId = userId
-    return next("router")
-  }
-
-  getRouter() {
-    const auth = express.Router()
-    auth.verifiedGet = (path) => {
-      auth.get(path, this.verifyJwt)
-    }
-    auth.verifiedDelete = (path) => {
-      auth.delete(path, this.verifyJwt)
-    }
-    auth.verifiedPost = (path) => {
-      auth.post(path, this.verifyJwt)
-    }
-
-    // Health check
-    auth.get("/v2/ping", this.noVerify)
-
-    // V1 endpoints
-    // TODO [#351]: Remove after refactor is done
```

Comment:
is #351 still relevant?

Reply:
Yup, will remove the V1 endpoints after all routes have been migrated